quiz.js

```diff
@@ -34,7 +36,16 @@ var dingHandle = setTimeout (writeDing(), 3000);
 var words = ['short', 'medium', 'long', 'interminable', 'lengthy'];
 
 var byLength = function (A, B) {
-  // your code here
+  if (A > B) {
```

Comment:
this code is perfect except for the fact that you need to compare the lengths of the strings